### ISSUE TITLE
[Package Issue]: Teamviewer.Teamviewer

### ISSUE THREAD

**Issue by GypsIT** [microsoft/winget-pkgs#116012]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

I'm trying to install Version 15.44.5 which results in the error "Installer hash does not match"

### Steps to reproduce

winget install teamviewer.teamviewer --source "winget" --scope=machine

### Actual behavior

Installation fails because the Installer hash does not match

### Expected behavior

Installation should work like a charm

### Environment

```raw
Windows Package Manager v1.5.1881
Windows: Windows.Desktop v10.0.19045.3208
Package: Microsoft.DesktopAppInstaller v1.20.1881.0
```

### Screenshots and Logs

![2023-08-09_17h58_04](https://github.com/microsoft/winget-pkgs/assets/52815895/9d9a03d2-b509-408a-8710-4b2abc7e0fe7)

**Comment by ImJoakim** [comment]
This should be resolved (https://github.com/microsoft/winget-pkgs/pull/115893). Can you try again and report back?

**Comment by GypsIT** [comment]
> This should be resolved (#115893). Can you try again and report back?

Hi ImJoaking, 

just tried it and it worked. Thank you very much

**Comment by simkoG** [comment]
The issue has reappeared in the new versions of TeamViewer.

```
winget install --id TeamViewer.TeamViewer
Found TeamViewer [TeamViewer.TeamViewer] Version 15.48.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.teamviewer.com/download/TeamViewer_Setup_x64.exe
  ██████████████████████████████  59.4 MB / 59.4 MB
Installer hash does not match; this cannot be overridden when running as admin
```

```
Windows: Windows.Desktop v10.0.22631.2715
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.21.3133.0
```
[WinGet-2023-11-25-16-29-46.034.log](https://github.com/microsoft/winget-pkgs/files/13464462/WinGet-2023-11-25-16-29-46.034.log)
